packages/cli/src/chat/agent/src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import { Chat } from '@shared/components/Chat';
-import { ThemeProvider } from '@shared/components/ui/theme-provider';
+import { Layout } from '@shared/components/Layout';
```

Comment:
i think its /layout here not /Layout, seems that't making the tests fail

Reply:
ohh, thanks